### ISSUE TITLE
Multiple results return from find_method

### ISSUE THREAD

**Issue by krnick** [quark-engine/quark-engine#56]
The `find_method` in [Object/apkinfo.py](https://github.com/quark-engine/quark-engine/blob/33ced423acdfccaf120f92c55a488806dec8d87f/quark/Objects/apkinfo.py#L32) uses a method called `find_methods` in Androguard, which uses regular expressions. After `find_method`, it will lead to one or more results return.

The period asterisk pattern (.*) is one of the most widely matched patterns used by Androguard in the `find_methods` method because it can match one or more patterns.

The situation of the problem is as below:

When we want to find the method `getActiveNetwork`, there is no such method in this APK, but the method `getActiveNetworkInfo` returned from the result and then crash the program.

**Comment by pulorsok** [comment]
- version: v20.01
- platform: Linux 5.3.0-40-generic #32-18.04.1-Ubuntu
- python version: python 3.7.5


## What steps will reproduce the bug?



Install Quark-Engine follow by the instruction and analysis an apk file with rules that I generated by the Android native API

`quark -a 14d9f1a92dd984d6040cc41ed06e273e.apk -r rule/ -s`

the bug occurs when the engine start to analyze with the rule below:
```
{
    "crime": {
        "action": "getActiveNetwork",
        "target": "getNetworkInfo"
    },
    "x1_permission": [
        "android.permission.ACCESS_NETWORK_STATE"
    ],
    "x2n3n4_comb": [
        {
            "class": "Landroid/net/ConnectivityManager",
            "method": "getActiveNetwork"
        },
        {
            "class": "Landroid/net/ConnectivityManager",
            "method": "getNetworkInfo"
        }
    ],
    "yscore": 1
}
```
## Current Behavior

when the engine passthrough the rule to analyze the apk file it shows up :
```
Traceback (most recent call last):
  File "/Users/pock/.local/share/virtualenvs/quark-engine-BYKEIt--/bin/quark", line 11, in <module>
    load_entry_point('quark-engine', 'console_scripts', 'quark')()
  File "/Users/pock/.local/share/virtualenvs/quark-engine-BYKEIt--/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/pock/.local/share/virtualenvs/quark-engine-BYKEIt--/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/pock/.local/share/virtualenvs/quark-engine-BYKEIt--/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/pock/.local/share/virtualenvs/quark-engine-BYKEIt--/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/pock/quark-engine/quark/cli.py", line 38, in entry_point
    data.run(rule_checker)
  File "/Users/pock/quark-engine/quark/Objects/xrule.py", line 269, in run
    if self.check_sequence(common_method, pre_0, pre_1):
  File "/Users/pock/quark-engine/quark/Objects/xrule.py", line 165, in check_sequence
    if s_func_val > f_func_val:
TypeError: '>' not supported between instances of 'int' and 'NoneType'
```

## Expected Behavior
The analysis progress should continue and complete show summary report

## The reason that caused the bug

Following the error message that shows the problem is at 'check_sequence()' method in 'xrule.py'.
it to search all methods in their upper function and missed one of the method, it then unable to  compare the sequence of the method call.

**Comment by krnick** [comment]
Closed by PR #76
